### PR TITLE
Remove beta guards for Epic #20168

### PR DESCRIPTION
**Pull request by teddyjtorres**
For Epic #20168

### CODE REVIEW

**Comment by teddyjtorres** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.client_fat.1,com.ibm.ws.security.oidc.client_fat.2,com.ibm.ws.security.oidc.client_fat.jaxrs,com.ibm.ws.security.oidc.client_fat.spnego,com.ibm.ws.security.oidc.server_fat.jaxrs.config,com.ibm.ws.security.oidc.server_fat.oidc,com.ibm.ws.security.oidc.server_fat.spnego,com.ibm.ws.security.oidc.social_fat.backchannelLogout,com.ibm.ws.security.oidc.social_fat.claimPropagation,com.ibm.ws.security.saml.sso_fat,com.ibm.ws.security.saml.sso_fat.2,com.ibm.ws.security.saml.sso_fat.3,com.ibm.ws.security.saml.sso_fat.config,com.ibm.ws.security.saml.sso_fat.config.mapToUserRegistry,com.ibm.ws.security.saml.sso_fat.endpoint.samlmetadata,com.ibm.ws.security.saml.sso_fat.jaxrs,com.ibm.ws.security.saml.sso_fat.jaxrs.config,com.ibm.ws.security.saml.sso_fat.logout,com.ibm.ws.security.saml.sso_fat.logout.httpServletRequest,com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout,com.ibm.ws.security.saml.sso_fat.logout.IDP_initiated,com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.commonTest.LibertyOP,com.ibm.ws.security.social_fat.delegated,com.ibm.ws.security.social_fat.LibertyOP.1,com.ibm.ws.security.social_fat.LibertyOP.2,com.ibm.ws.security.social_fat.multiProvider,com.ibm.ws.security.social_fat.okdServiceLogin,com.ibm.ws.security.social_fat.OpenShift

File: dev/com.ibm.ws.security.oidc.client_fat.jaxrs/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.jaxrs.op/jvm.options [hunk 1]
```diff
@@ -8,4 +8,4 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
--Dcom.ibm.ws.beta.edition=true
+-Dcom.ibm.ws.beta.edition=false
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.client_fat.jaxrs/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.jaxrs.op/jvm.options:11]: This works just fine, but we'll want to come back and just take out the property entirely.

File: dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/internal/OidcClientImpl.java [hunk 2]
```diff
@@ -596,15 +596,7 @@ String getHeader(HttpServletRequest req, String headerName) {
...
-            provider = getProviderConfigCurrent(oidcClientConfigs, reqProviderHint, req);
-        } else {
-            provider = getProviderConfigBeta(reqProviderHint, req);
-        }
-
-        return provider;
+        return getProviderConfig(reqProviderHint, req);
     }
 
     protected String getProviderConfigCurrent(Iterator<OidcClientConfig> oidcClientConfigs,
```
Comment (ayoho) [inline dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/internal/OidcClientImpl.java:602]: Can come back and take this method out entirely.

**Review by ayoho — APPROVED** [review]

**Comment by teddyjtorres** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.client_fat.1,com.ibm.ws.security.oidc.client_fat.2,com.ibm.ws.security.oidc.client_fat.jaxrs,com.ibm.ws.security.oidc.client_fat.spnego,com.ibm.ws.security.oidc.server_fat.jaxrs.config,com.ibm.ws.security.oidc.server_fat.oidc,com.ibm.ws.security.oidc.server_fat.spnego,com.ibm.ws.security.oidc.social_fat.backchannelLogout,com.ibm.ws.security.oidc.social_fat.claimPropagation,com.ibm.ws.security.saml.sso_fat,com.ibm.ws.security.saml.sso_fat.2,com.ibm.ws.security.saml.sso_fat.3,com.ibm.ws.security.saml.sso_fat.config,com.ibm.ws.security.saml.sso_fat.config.mapToUserRegistry,com.ibm.ws.security.saml.sso_fat.endpoint.samlmetadata,com.ibm.ws.security.saml.sso_fat.jaxrs,com.ibm.ws.security.saml.sso_fat.jaxrs.config,com.ibm.ws.security.saml.sso_fat.logout,com.ibm.ws.security.saml.sso_fat.logout.httpServletRequest,com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout,com.ibm.ws.security.saml.sso_fat.logout.IDP_initiated,com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.commonTest.LibertyOP,com.ibm.ws.security.social_fat.delegated,com.ibm.ws.security.social_fat.LibertyOP.1,com.ibm.ws.security.social_fat.LibertyOP.2,com.ibm.ws.security.social_fat.multiProvider,com.ibm.ws.security.social_fat.okdServiceLogin,com.ibm.ws.security.social_fat.OpenShift

**Comment by teddyjtorres** [comment]
#build

**Pull request merged**